website/client/src/components/payments/sendGiftModal.vue

```diff
@@ -0,0 +1,638 @@
+<template>
+  <b-modal
+    id="send-gift"
+    :hide-footer="true"
+    :hide-header="true"
+    size="md"
+    @hide="onHide()"
+  >
+    <div>
+      <!-- header -->
+      <div
+        class="modal-close"
+        @click="close()"
+      >
+        <div
+          class="icon-close"
+          v-html="icons.closeIcon"
+        >
+        </div>
+      </div>
+      <div>
+        <h2 class="d-flex flex-column mx-auto align-items-center">
+          {{ $t('sendAGift') }}
+        </h2>
+      </div>
+      <!-- user avatar -->
+      <div
+        v-if="userReceivingGift"
+        class="modal-body"
+      >
+        <avatar
+          :member="userReceivingGift"
+          :hideClassBadge="true"
+          class="d-flex flex-column mx-auto align-items-center"
+        />
+        <div class="avatar-spacer"></div>
+        <div
+          class="d-flex flex-column mx-auto align-items-center display-name">
+          <!-- user display name and username -->
+            <user-link
+              :user-id="displayName"
+              :name="displayName"
+              :backer="userBacker"
+              :contributor="userContributor"
+              :class="display-name"
+              />
```

Comment:
There's a bunch of weird indentation in this file, first instance here but it's throughout--would it work for me to just proofread and commit, you think?